lib/std/os/linux/sparc64.zig

```diff
@@ -21,6 +21,23 @@ pub fn syscall_pipe(fd: *[2]i32) usize {
     );
 }
 
+pub fn syscall_fork() usize {
```

Comment:
Do you have any docs explaining how the results are returned?
I guess the parent and the child pids are returned in `o0` and `o1`, but what's the `dec`/`and` doing in the error path?

Reply:
https://elixir.bootlin.com/linux/v5.9.11/source/arch/sparc/kernel/process_64.c#L580
The call chain is something like `sparc_fork`->`_do_fork`->`copy_process`->`copy_thread`.
%o1 will be 0 on the parent's side, and 1 on the child's side, hence the dec/and dance to clear %o0 on the child process.

Reply:
> %o1 will be 0 on the parent's side, and 1 on the child's side, hence the dec/and dance to clear %o0 on the child process.

Please add a brief comment explaining this weird 1/0 convention.

Reply:
Done.